classes/wp-background-process.php

```diff
@@ -251,6 +251,30 @@ protected function resumed() {
 		do_action( $this->identifier . '_resumed' );
 	}
 
+	/**
+	 * Is queued?
+	 *
+	 * @return bool
+	 */
+	public function is_queued() {
+		$batch = $this->get_batch();
+
+		if ( empty( $batch ) ) {
+			return false;
+		}
+
+		return true;
+	}
+
+	/**
+	 * Is the tool currently active, e.g. starting, working, paused or cleaning up?
+	 *
+	 * @return bool
+	 */
+	public function is_active() {
+		return $this->is_queued() || $this->is_processing() || $this->is_paused() || $this->is_cancelled();
```

Comment:
`is_cancelled` still considered active?

Reply:
Yep, there may be a batch still being processed, once that finishes up the cancelled status will be handled and the queue cleared etc.